### PR TITLE
Issue #11 from the mailine project. Added support for outbound/egress…

### PR DESCRIPTION
**Pull request by jcuzens**
#11 … rules.

### CODE REVIEW

**Comment by jcuzens** [comment]
Not working on CSV. Reopening to fix the CSV.

File: ec2_security_groups_dumper/main.py [hunk 1]
```diff
@@ -215,6 +224,7 @@ def csv(self):
         - id
         - name
         - description
+	- rules_direction
```
Comment (lra) [inline ec2_security_groups_dumper/main.py]: bad indent

File: ec2_security_groups_dumper/main.py [hunk 2]
```diff
@@ -275,31 +286,45 @@ def _get_rules_from_aws(self):
                 group_dict['rules'] = list()
 
             for rule in group.rules:
-                rule_dict = dict()
-                rule_dict['ip_protocol'] = rule.ip_protocol
-                rule_dict['from_port'] = rule.from_port
-                rule_dict['to_port'] = rule.to_port
+		rule_dict = self._build_rule( rule )
```
Comment (lra) [inline ec2_security_groups_dumper/main.py]: can you use space for indentation?

File: ec2_security_groups_dumper/main.py [hunk 3]
```diff
@@ -275,31 +286,45 @@ def _get_rules_from_aws(self):
...
-                        grant_dict['cidr_ip'] = grant.cidr_ip
+                group_dict['rules'].append(rule_dict)
 
-                    rule_dict['grants'].append(grant_dict)
+            if group.rules_egress:
+                for rule in group.rules_egress:
+                    rule_dict = self._build_rule( rule )
+                    rule_dict['direction']="EGRESS"
+                        
+                    group_dict['rules'].append(rule_dict)
```
Comment (lra) [inline ec2_security_groups_dumper/main.py]: `group_dict['rules']` is undefined if you don't have any VPC rule.
Can you test your PR in both scenario?

**Comment by lra** [comment]
Thanks!

**Pull request merged**